src/app/services/user.service.ts

```diff
@@ -113,7 +113,16 @@ export class UserService {
       description: data.description,
       real_name: data.real_name
     };
-    return this.http.put(this.USER_URL + '/' + data.userId, payload).pipe(
+    return this.http.put(this.ADMIN_URL + '/' + data.userId, payload).pipe(
+      map(response => response),
+      catchError(error => throwError(error)));
+  }
+
+  updateMe(data) {
+    const payload: any = {
+      real_name: data.real_name
```

Comment:
test 
* a normal user cannot update role, access method, description of herself/ others.
* disabled user cannot log in
* deleted user cannot log in

and super admin (user id 1) is always there, cannot be deleted, role cannot be changed and only herself can update profile attributes (even not other admin)

Reply:
Tested all of the above, it seems Ok.